examples/stream/stream/create.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2017 Caicloud Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package stream
+
+import (
+	"bytes"
+	"context"
+	"io"
+)
+
+func Stream(ctx context.Context) (io.Reader, error) {
```

Comment:
The framework is responsible to read and stream the data, and the 'interface' we provide to framework user is `definition.Data`?

Reply:
Yes. Whatever the type of the returned value is, for users, it's `Data`.